### PR TITLE
FAQ: New Automation section

### PR DESCRIPTION
**Pull request by psiinon**
We might want to move the new Automation section up, but that would mean changing lots of other FAQs..

I've tested the variable setting scripts on macOS and Windows

<details><summary>Screenshots</summary>
<p>

Screenshot 2024-12-12 at 13-32-08 ZAP – What is the Best Way to Automate ZAP

Screenshot 2024-12-12 at 13-31-57 ZAP – How Can I set Variables in the Automation Framework

</p>
</details>

### CODE REVIEW

**Comment by psiinon** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [516854e1\-6da8\-48bb\-aa75\-62048cbb3bc1](https://eu.ast.checkmarx.net/projects/f9121e40-c915-4b13-bb15-36d946943fd8/scans?id=516854e1-6da8-48bb-aa75-62048cbb3bc1&branch=psiinon%2Ffaq%2Fauto)
### New Issues
Severity|Issue|Source File / Package|Checkmarx Insight
---|---|---|---
HIGH|[CVE-2024-52798](https://devhub.checkmarx.com/cve-details/CVE-2024-52798)|Npm\-path\-to\-regexp\-0\.1\.10|[Vulnerable Package](https://eu.ast.checkmarx.net/results/f9121e40-c915-4b13-bb15-36d946943fd8/516854e1-6da8-48bb-aa75-62048cbb3bc1/sca?internalPath=%2Fpackages%2FNpm-path-to-regexp-0.1.10%2FpackageDetailsGql)
### Fixed Issues
Severity|Issue|Source File / Package
---|---|---
HIGH|[Cx89601373-08db](https://devhub.checkmarx.com/cve-details/Cx89601373-08db)|Npm\-debug\-3\.2\.7
HIGH|[Cx89601373-08db](https://devhub.checkmarx.com/cve-details/Cx89601373-08db)|Npm\-debug\-2\.6\.9
LOW|Unpinned Actions Full Length Commit SHA|/zap\-full\-scan\.yml: [17](https://github.com/psiinon/zaproxy-website/blob/faq/auto//.github/workflows/zap-full-scan.yml#L17)
LOW|Unpinned Actions Full Length Commit SHA|/zap\-scan\.yml: [17](https://github.com/psiinon/zaproxy-website/blob/faq/auto//.github/workflows/zap-scan.yml#L17)

File: site/content/faq/how-can-i-set-variables-in-the-automation-framework.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: "How Can I set Variables in the Automation Framework?"
+type: faq
+category: Automation
+weight: 8
+---
+
+The [Automation Framework](/docs/automate/automation-framework/) supports 
+[variables](/docs/desktop/addons/automation-framework/environment/#variables), which includes environmental variables.
+You can use these to set variables referenced in your plan from the command line, including secrets such as credentials.
```
Comment (kingthorin) [inline site/content/faq/how-can-i-set-variables-in-the-automation-framework.md:10]: to reduce the occurrences of "variables" could we say "...to set values referenced....."?

Comment (psiinon) [reply site/content/faq/how-can-i-set-variables-in-the-automation-framework.md]: Done

**Review by kingthorin — APPROVED** [review]

File: site/content/faq/how-can-i-set-variables-in-the-automation-framework.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+title: "How Can I set Variables in the Automation Framework?"
+type: faq
+category: Automation
+weight: 8
+---
+
+The [Automation Framework](/docs/automate/automation-framework/) supports 
+[variables](/docs/desktop/addons/automation-framework/environment/#variables), which includes environmental variables.
```
[the remark this answers is not in the document]
Comment (psiinon) [reply site/content/faq/how-can-i-set-variables-in-the-automation-framework.md]: done

**Review by ricekot — APPROVED** [review]
Thanks!

**Pull request merged**